docs/views/examples/pass-data/index.html

```diff
@@ -82,10 +82,32 @@ <h3 class="govuk-heading-s">
       <h3 class="govuk-heading-s">
         Setting default data
       </h3>
+      <p>
+        You can set default data in your prototype. This will appear without the
+        user having to enter anything. For example, if you want to prototype a
+        journey where a user previously saved their responses and returned to it
+        later to review or update the information already in the prototype.
+      </p>
+      <p>
+        If the user changes this data in the prototype, their new answers will
+        be saved.
+      </p>
+      <p>
+        Add default data to this file:
+      </p>
+      <pre class="app-code" tabindex="0"><code>app/data/session-data-defaults.js</code></pre>
+      <p>
+        For example to set default data for the inputs for 'First name' and
+        'Over 18' in the examples above, you would have this in your
+        session-data-defaults.js file:
+      </p>
+      <pre class="app-code" tabindex="0"><code>
+module.exports = {
 
-      <p>You can set default values in this file in your prototype folder:
+  'first-name': 'Amina',
+  'over-18': 'yes'
 
```

Comment:
Not sure about the extra blank lines, they look a bit weird to me?

```
module.exports = {

  'first-name': 'Amina',
  'over-18': 'yes'

}
```

https://govuk-protot-update-def-niejz0.herokuapp.com/docs/examples/pass-data

Reply:
the lines before first name and after over-18? They're there already in the default session data file, I think they make `// Insert values here` stand out more

Reply:
Makes sense in the template, not sure it has to be in the example though? I'm not fussed either way though.